internal/datastore/revisions/hlcrevision_test.go

```diff
@@ -108,7 +108,7 @@ func TestInexactFloat64(t *testing.T) {
 			rev, err := HLCRevisionFromString(tc)
 			require.NoError(t, err)
 
-			require.Equal(t, floatValue, rev.InexactFloat64())
+			require.InEpsilon(t, floatValue, rev.InexactFloat64(), 1e-9)
```

Comment:
We don't want to give a % of acceptable deviation here. This is parsing a hybrid logical clock from a string, and we expect the values to be precisely what is provided in the table tests.